exercises/isbn-verifier/description.md

```diff
@@ -9,27 +12,35 @@ The program should allow for ISBN-10 without the separating dashes to be verifie
 
 ## ISBN
 
-Let's take a random ISBN-10 number, say `3-598-21508-8` for this.
+The ISBN-10 format is 9 digits (0 to 9) plus one check character (either a digit or an X only). In the case the check character is an X, this represents the value '10'. These may be communicated with or without hyphens, and can be checked for their validity by the following formula:
+
+```
+(x1 * 10 + x2 * 9 + x3 * 8 + x4 * 7 + x5 * 6 + x6 * 5 + x7 * 4 + x8 * 3 + x9 * 2 + x10 * 1) mod 11 == 0
+```
+
+If the result is 0, then it is a valid ISBN-10, otherwise it is invalid.
+
+## Example
+
+Let's take a random ISBN-10 number, for instance `3-598-21508-8`.
 The first digit block indicates the group where the ISBN belongs. Groups can consist of shared languages, geographic regions or countries. The leading '3' signals this ISBN is from a german speaking country.
 The following number block is to identify the publisher. Since this is a three digit publisher number there is a 5 digit title number for this book.
 The last digit in the ISBN is the check digit which is used to detect read errors.
 
-The first 9 digits in the ISBN have to be between 0 and 9.
-The check digit can additionally be an 'X' to allow 10 to be a valid check digit as well.
-
-A valid ISBN-10 is calculated with this formula `(x1 * 10 + x2 * 9 + x3 * 8 + x4 * 7 + x5 * 6 + x6 * 5 + x7 * 4 + x8 * 3 + x9 * 2 + x10 * 1) mod 11 == 0`
 So for our example ISBN this means:
+```
 (3 * 10 + 5 * 9 + 9 * 8 + 8 * 7 + 2 * 6 + 1 * 5 + 5 * 4 + 0 * 3 + 8 * 2 + 8 * 1) mod 11 = 0
+```
 
-Which proves that the ISBN is valid.
+Since the result is 0, this proves that our ISBN is valid.
```

Comment:
Good clarification 👍

Reply:
Should there be one `=` or two at the end of the line?  Line 18 only has one.

Reply:
See previous comment re: line 9.